crates/trippy-core/Cargo.toml

```diff
@@ -0,0 +1,61 @@
+[package]
+name = "trippy-core"
+description = "A network tracing library"
+version.workspace = true
+authors.workspace = true
+documentation.workspace = true
+homepage.workspace = true
+repository.workspace = true
+readme.workspace = true
+license.workspace = true
+edition.workspace = true
+rust-version.workspace = true
+keywords = ["traceroute", "ping", "icmp"]
+categories = ["network-programming"]
+
+[dependencies]
+derive_more.workspace = true
+thiserror.workspace = true
+tracing.workspace = true
+itertools.workspace = true
+arrayvec = { version = "0.7.4", default-features = false }
+socket2 = { version = "0.5.7", features = [ "all" ] }
+bitflags = "2.5.0"
+
+[target.'cfg(unix)'.dependencies]
+nix = { version = "0.28.0", default-features = false, features = [ "user", "poll", "net" ] }
+
+[target.'cfg(windows)'.dependencies]
+paste = "1.0.15"
+widestring = "1.0.2"
+windows-sys = { version = "0.52.0", features = [
+    "Win32_Foundation",
+    "Win32_Networking_WinSock",
+    "Win32_System_IO",
+    "Win32_NetworkManagement_IpHelper",
+    "Win32_NetworkManagement_Ndis",
+    "Win32_System_IO",
+    "Win32_System_Threading",
+    "Win32_Security",
+] }
+
+[dev-dependencies]
+anyhow.workspace = true
+serde.workspace = true
+test-case.workspace = true
+serde_yaml.workspace = true
+hex-literal = "0.4.1"
+rand = "0.8.5"
+tokio = { version = "1.37.0", features = [ "full" ] }
+tokio-util = "0.7.11"
+ipnetwork = "0.20.0"
+mockall = "0.12.1"
+tracing-subscriber = "0.3.18"
+
+# see https://github.com/meh/rust-tun/pull/74
+[target.'cfg(any(target_os = "macos", target_os = "linux", target_os = "windows"))'.dev-dependencies]
```

Comment:
Is there a reason not to just include this in the dev dependencies? It sees to be specified for all targets except WASM that I can think of.

Reply:
From memory it failed on some build targets in CI (netbsd and freebsd I think).